emissions.md

```diff
@@ -35,50 +47,65 @@ CO<sub>2</sub> emissions from gas flaring are not currently included in GCAM.
+
+* Additional information on fluorinated gases is from Guus Velders.
+
+##### Calibration year differences between CEDS and GCAM
+
+Figures 1 and 2 compare historical emissions from CEDS with the emissions from GCAM after initialization . Figure 1 compares global emissions by species and Figure 2 presents a scatter plot comparing emissions by species and year for each region.  As seen in the figures, emissions translate mostly correctly for all species of gases. 
+
+![Figure 1 : Comparison of global CEDS and GCAM emissions in calibration years](images/1a. Global_comparison_of_NonCO2Global.png)
+
+
+![Figure 2 : Comparison of regional CEDS and GCAM emissions in all calibration years](images/1b. Scatterplot comparing emissionsGlobal.png)
+
+However, there are two reasons for differences. The first is that of deforestation emissions from protected land which get zeroed out in GCAM since protected land is held constant in the calibration years. Also deforestation emissions in the final base year are initialized using deforestation coefficients calculated on the basis of deforestation over a 5 year period (2000 and 2005). This leads to a difference in the total deforestation emissions from the CEDS inventory when compared to the numbers initialized in  GCAM. 
+
+Second, the emissions from resource production are different between CEDS and GCAM in calibration years.This happens because the production of fossil fuels in GCAM is broken down by 'vintages' while the emissions factors are calculated in each model year based on total production in each year. When GCAM calculates the emissions, the emissions are calculated for each vintage using the emission factor from that year (for example, the 1975 vintage production will use the 1975 emission factor and the 2015 vintage will use the 2015 emission factor). Since older vintages will have higher emissions factors, the emissions in GCAM will be higher compared to CEDS inventory emissions.
+
+#### Modeling approach
 
-Non-CO<sub>2</sub> emissions, both GHGs & air pollutants, originate from many sources and can be controlled using multiple abatement technologies.
 Modeling non-CO<sub>2</sub> abatement at the process level would require too much detail for the scales at which GCAM operates. We, therefore, use parameterized functions for future emissions controls (for air pollutants) and Marginal Abatement Cost (MAC) curves (for GHGs) to change emission factors over time. The emissions controls, which reduce emissions factors as a function of per-capita GDP in each region and time period, are based on the general understanding that pollutant control technologies are deployed as incomes rise (e.g., [Smith et al. 2005](#smith2005), although the functional form used in GCAM 5 is different than that in this reference). The MAC curves for GHGs are mapped directly to GCAM's technologies from the EPA's 2013 report on non-CO<sub>2</sub> greenhouse gas mitigation ([EPA 2013](#epa2013)).
```

Comment:
change to EPA’2019 report on non-CO2 greenhouse gas mitigation [https://www.epa.gov/global-mitigation-non-co2-greenhouse-gases/global-non-co2-greenhouse-gas-emission-projections](url)